src/app/cube/browse/components/filter-section/filter-section.component.html

```diff
@@ -4,7 +4,7 @@
         <span *ngIf="collapsed"><i class="fas fa-angle-down"></i></span>
         <span *ngIf="!collapsed"><i class="fas fa-angle-up"></i></span>
     </div>
-    <div class="filter-wrapper" *ngIf="!collapsed">
+    <div class="filter-wrapper" *ngIf="!collapsed || hasActiveStatus()">
```

Comment:
This will make it unable to collapse the filter if one is selected.  It should only be forced open on app load/reload and one is active.